test/runtests.jl

```diff
@@ -1,10 +1,24 @@
+# path=pwd()
+# if path[end-6:end] != "MatLang"
+# 	error("cd(to the MatLang path)")
+# end
+
+push!(LOAD_PATH, ".")
```

Comment:
Is this necessary?

Reply:
That is based on this: https://www.oreilly.com/library/view/julia-10-programming/9781788998369/61241ac3-23ba-443c-b50d-55f26f16493d.xhtml

Reply:
I don't think this is necessary.